### PR TITLE
moved reference for the main figures to refer to the analysis repo instead of the manuscript repo

### PR DESCRIPTION
**Pull request by nrosed**
I moved main figure references from referring to figures within the manuscript repo to now reference the analysis repo. I temporarily used the manuscript repo when I needed to quickly rebuild the manuscript, but had pending PRs in the analysis repo.

I also changed one example name in the schematic figure (first panel of figures 2,3 and 4) from S. Hyland to Laarni Hyland.

There **_should_** be no other changes in the content of the manuscript.

### CODE REVIEW

File: content/04.results.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ The left annotation bar titled "median_M_C" is the median difference across all
 The "Class" annotation column denotes the binarized set definition of each country, either "Cited" or "Mentioned".
 Panel E shows the top 15 words extracted from articles mentioning the countries depicted in Panel D, with the largest proportional frequency between the two defined country sets.
 The width of the bar depicts the log10(Frequency in Mentions +1 / Frequency of Citations +1).
-](https://github.com/nrosed/nature_news_manuscript/raw/main/content/images/fig4_tmp/fig4_main.png "Figure 4"){#fig:fig4 width=6in}
+](https://github.com/nrosed/nature_news_disparities/raw/main/figure_notebooks/tmp_files/fig4_tmp/fig4_main.png "Figure 4"){#fig:fig4 width=6in}
```
Comment (cgreene) [inline content/04.results.md:217]: should these definitely be named tmp_files? just my only question - if so 🚀

**Review by cgreene — APPROVED** [review]
one small question

File: content/04.results.md (hunk 1 above)
Comment (nrosed) [reply content/04.results.md]: haha, yes good point. changed from tmp_files -> manuscript_figs in https://github.com/greenelab/nature_news_manuscript/pull/13/commits/dae2c51efe6c088d3eda3b85c9fee20361dd8d02

Comment (cgreene) [reply content/04.results.md]: awesome! 👍

**Pull request merged**